features/facebookImport/src/context/importer-context.jsx

```diff
@@ -149,6 +149,9 @@ export const ImporterProvider = ({ children }) => {
             if (history.location.state) {
                 changeNavigationState(history.location.state);
             }
+            if (location.pathname === "/report/learnmore") {
+                history.goBack();
+            }
```

Comment:
Uh, that's a bit hacky I would say :D But I guess it works!

I think we need some kind of `goBackTo(URL)` or something for this case, since if we were to navigate _forward_ we would get that circular back navigation... (@RichardSto - not the right place to fix, or probably discuss this, but FYI :D)